Sources/ComposableArchitecture/Documentation.docc/Tutorials/MeetTheComposableArchitecture/02-Navigation/04-NavigationStacks/02-04-01-code-0005.swift

```diff
@@ -1,4 +1,5 @@
 import SwiftUI
+import ComposableArchitecture
```

Comment:
Do you think you have time to alphabetize these? For example:

```suggestion
import ComposableArchitecture
import SwiftUI
```

Reply:
Sure! I changed these things. Please check my commit. Thanks.